ontology/yaml/resources/states/states.yaml

```diff
@@ -69,5 +69,10 @@ POSITIONA: "Automatic Transfer switch position 0"
 POSITIONB: "Automatic Transfer switch position 1"
 POSITIONC: "Automatic Transfer switch position 2"
 
-
-
+#FOR UPS
+BATTERY: "Operation mode or input power is via Battery"
+SYNCHRONOUS: "Mode of operation of inverter"
+ASYNCHRONOUS: "Mode of operation of inverter"
```

Comment:
please differentiate between lines 74 and 75, they have the same definition: what do synchronous and asynchronous mean?

Reply:
Added the definition in detail.